lib/core/network/networking_manager.dart

```diff
@@ -4,15 +4,19 @@ import 'package:hypha_wallet/core/network/api/endpoints.dart';
 import 'package:pretty_dio_logger/pretty_dio_logger.dart';
 
 class NetworkingManager extends DioForNative {
+  static final bool _isDebugNetworking = false;
+
   NetworkingManager(String baseUrl) : super() {
-    // Add Logs Interceptor
-    interceptors.add(PrettyDioLogger(
-        requestHeader: false,
-        requestBody: true,
-        responseBody: true,
-        responseHeader: false,
-        error: true,
-        compact: true));
+    if (_isDebugNetworking) {
```

Comment:
I think it's a compromise - I never want to see these, it clogs my debug logs. 

Since I mainly use print statements to debug code, I can't have anything in there that's not relevant to what I am doing.

So made a flag for you - you can set it to true, or to kDebugMode, I can set it to false. 

Console should only show
- errors
-  things I am working on (but will be removed before merging)

anything else is fluff and distracting / making debugging harder as I am concerned. 

I mean I love these super detailed network logs, and I'll certainly turn them on when I debug anything network related. But don't want them around all the time.

Reply:
Haha Noob! there is a debug feature in your IDE! :) 

But yeah, i understand what you mean.  Im cool with this.